books/free-programming-books-ru.md

```diff
@@ -132,6 +133,11 @@
 * [Просто о Vim](http://rus-linux.net/MyLDP/BOOKS/Vim/prosto-o-vim.pdf) - Swaroop (PDF)
 
 
+### Arduino
+
+* [Science, Programming, Art and Radioelectronics Club (SPARC)](https://github.com/artyom-poptsov/SPARC) - Artyom V. Poptsov (PDF) (CC BY-SA)
```

Comment:
```suggestion
* [Автомато-программато-компарадио-кружок](https://github.com/artyom-poptsov/SPARC/blob/master/README.ru.org) - Artyom V. Poptsov (PDF) (CC BY-SA)
```

The russian list should use the russian title. Also, can't have the exact same link on two lists!

Reply:
Okay, thanks!  Linking to the Russian `README` is the best option indeed.